driver.schema.json

```diff
@@ -115,6 +115,52 @@
           }
         }
       }
+    },
+    "status_info": {
+      "type": "object",
+      "title": "Mapping of status info key to content and level",
+      "additionalProperties": {
+        "type": "object",
+        "properties": {
+          "content": {
+            "type": "string",
+            "title": "Content for the status info",
+            "examples": ["Low Oil Pressure", "High Temperature"]
+          },
+          "register": {
+            "type": "integer",
+            "title": "Modbus register for the status info"
+          },
+          "start_bit": {
+            "type": "integer",
+            "title": "Start bit (within the register) for the status info, first=1"
+          },
+          "length_bits": {
+            "type": "integer",
+            "title": "Length in bits for the status info"
+          },
+          "order": {
+            "type": "string",
+            "title": "Bit order",
+            "description": "Bit order within the register, msr=most significant bit first, lsr=least significant bit first",
```

Comment:
Not sure if the description is correct

Reply:
Also we'll add a `status_info_common` top-level object (see Confluence example)